crypto/s2n_ecc_evp.c

```diff
@@ -89,6 +89,13 @@ const struct s2n_ecc_named_curve s2n_ecc_curve_x25519 = {
 const struct s2n_ecc_named_curve s2n_ecc_curve_x25519 = {0};
 #endif
 
+const struct s2n_ecc_named_curve s2n_unsupported_curve = {
```

Comment:
This is only needed for testing right? Is there a way to move this closer to the test its needed in? If not, I'd probably add a comment stating why it exists

Reply:
Correct, only needed for testing. The actual declaration needs to be in this file bc it uses a static method, but I added a comment and moved the declaration to the test file where it's needed.